src/main/java/stream/flarebot/flarebot/commands/music/SkipCommand.java

```diff
@@ -189,8 +200,12 @@ public String toString() {
     }
 
     private void addButtons(ButtonGroup buttons, TextChannel channel) {
+        VoiceChannel vc = channel.getGuild().getSelfMember().getVoiceState().getChannel();
         buttons.addButton(new ButtonGroup.Button(355776056092917761L, (user, message) -> {
-
+            if(!channel.getGuild().getMember(user).getVoiceState().inVoiceChannel() || !channel.getGuild().getMember(user).getVoiceState().getChannel().equals(vc)) {
```

Comment:
Don't check if channels are equal, compare getIdLong

Reply:
Doesn't #equals() do that anyway?

Reply:
Equals compares the whole object, JDA doesn't override it to make it do this so doing it ourselves would be most efficient.